css/style.css

```diff
@@ -0,0 +1,188 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+main {
+  font-family: Lato, sans-serif;
+  font-size: 22px;
+}
+
+/* easy attributes */
+.dflex {
+  display: flex;
+}
+
+.flex1 {
+  flex: 1;
+}
+
+.background-gradient {
+  background-image: linear-gradient(rgb(114, 111, 111) 50%, rgb(97, 95, 95) 40%, rgb(97, 95, 95));
+}
+
+.z-index-top {
+  z-index: 2;
+}
+
+.z-index-bot {
+  z-index: 0;
+}
+
+.flex-ietm-1-8 {
+  flex: 1;
+  min-width: 50px;
+}
+
+.flex-item-9 {
+  flex: 1;
+}
```

Comment:
Group this selector with the selector on line 17, both have the same style definition. As an alternative, remove one and just use the other.

Reply:
Thank you Andres, 

### to the next review:
##### We have:
* removed flex-item-9 and used flex1 in its stead.
* increased size of the height of background div to match the height of the picture in the original (approx the size of his bottle under text)